test/topology/util.py

```diff
@@ -12,11 +12,12 @@
 import operator
 import pytest
 import time
-from cassandra.cluster import Session  # type: ignore # pylint: disable=no-name-in-module
```

Comment:
Why does this PR only `Ref`s #17409, and not `Fixes`?

The commit message should contain `Ref` (or `Fixes`) too. AFAIK, in the case of a 1-commit PR, the commit is cherry-picked and it's message is the one that remains, not the cover letter.

Reply:
Afaik if I would use `Fixes`, merging this PR would close the issue, but I will also open PR for the topology recovery test (also mentioned in the issue).

> The commit message should contain Ref (or Fixes) too. AFAIK, in the case of a 1-commit PR, the commit is cherry-picked and it's message is the one that remains, not the cover letter.

TIL

Reply:
> but I will also open PR for the topology recovery test (also mentioned in the issue).
I see, it makes sense then.